mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -4171,12 +4171,13 @@
 											},
 											{
 												"key": "Content-Type",
-												"value": "application/vnd.api+json"
+												"value": "application/vnd.api+json",
+												"type": "text"
 											}
 										],
 										"body": {
 											"mode": "raw",
-											"raw": "{\n  \"data\": {\n    \"type\": \"providers\",\n    \"attributes\": {\n      \"providerToken\": {\n        \"value\": null\n      },\n      \"proxy\": {\n        \"id\": \"<n>\"\n      }\n    }\n  }\n}"
+											"raw": "{\r\n  \"data\": {\r\n    \"id\": \"19\",\r\n    \"type\": \"providers\",\r\n    \"attributes\": {\r\n      \"name\": \"EBSCO\",\r\n      \"packagesTotal\": 625,\r\n      \"packagesSelected\": 19,\r\n      \"supportsCustomPackages\": false,\r\n      \"proxy\": {\r\n        \"id\": \"<n>\",\r\n        \"inherited\": false\r\n      }\r\n    }\r\n  }\r\n}\r\n"
```

Comment:
curious about this carriage return that got appended just for this payload. otherwise looks good.

Reply:
Might be incorrect saving when exporting. I'l fix it